docs/control-structures.rst

```diff
@@ -20,6 +20,45 @@ there is in C and JavaScript, so ``if (1) { ... }`` is *not* valid Solidity.
 
 .. index:: ! function;call, function;internal, function;external
 
+Input Parameters and Return Parameters
+======================================
+
+Like in Javascript and in C, functions may take parameters as input;
+unlike in Javascript and C, they may also return arbitrary number of
+parameters as output. The input parameters are declared type followed
+by name (though unused input parameters do not need names).
```

Comment:
This sentence is about input parameters.  For output parameters, see the next paragraph.